src/Tizen.NUI/src/public/Utility/DpTypeConverter.cs

```diff
@@ -0,0 +1,83 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System.ComponentModel;
+using System.Globalization;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// DpTypeConverter class to convert types.
+    /// </summary>
+    /// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class DpTypeConverter : GraphicsTypeConverter
+    {
+        /// <summary>
+        /// Converts script to px
+        /// </summary>
+        /// <returns>Pixel value that is converted from input string</returns>
+        /// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override float ConvertScriptToPixel(string scriptValue)
+        {
+            float convertedValue = 0;
+            if (scriptValue != null)
+            {
+                if (scriptValue.EndsWith("dp"))
+                {
+                    convertedValue = ConvertToPixel(float.Parse(scriptValue.Substring(0, scriptValue.LastIndexOf("dp")), CultureInfo.InvariantCulture));
+                }
+                else if (scriptValue.EndsWith("px"))
+                {
+                    convertedValue = float.Parse(scriptValue.Substring(0, scriptValue.LastIndexOf("px")), CultureInfo.InvariantCulture);
+                }
+                else
+                {
+                    if (!float.TryParse(scriptValue, NumberStyles.Any, CultureInfo.InvariantCulture, out convertedValue))
+                    {
+                        NUILog.Error("Cannot convert the script {scriptValue}\n");
+                        convertedValue = 0;
+                    }
+                }
+            }
+            return convertedValue;
+        }
+
+        /// <summary>
+        /// Converts dp type to px
+        /// </summary>
+        /// <returns>Pixel value that is converted by the the display matric</returns>
+        /// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override float ConvertToPixel(float value)
+        {
+            return value * (GraphicsTypeManager.Dpi / GraphicsTypeManager.DefaultDpi);
+        }
+
+        /// <summary>
+        /// Converts px to dp type
+        /// </summary>
+        /// <returns>An converted value from pixel</returns>
+        /// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override float ConvertFromPixel(float value)
+        {
+            return value * (GraphicsTypeManager.DefaultDpi / GraphicsTypeManager.Dpi);
```

Comment:
int / int 라서 정밀도가 많이 떨어진 결과 값이 예상됩니다.
```suggestion
            return value * (GraphicsTypeManager.DefaultDpi / (float)GraphicsTypeManager.Dpi);
```